### DISCUSSION TITLE
Channel Waiting close for ever after server migration.

### DISCUSSION THREAD

**Discussion by Nisaba** [lightningnetwork/lnd#8718] [Troubleshooting]
Hi,

I run LND under btcpay and I had to move from a server to another.
So, I migrate the dockers and ensured that LN service was down in the new server.

About the LN network, here is what I did, not sure it was the right process :

- Backup channel states in the old server
- Stop the LN service in the old server
- Start the LN service in the new server
- Restore the channel states in the new server

So, all of my 4 channels are going to closing status ! Not sure why, but I suspect an IP conflict...
1 channel went to force closed status and I get funds back onchain.
But the 3 others channel stay in "Waiting close" status for ever...
What can I do to get funds back on chain ?

Here are the nodes:
4343a70d70e0cbd673f30e4d57e1cc29b4c8bb751bca6f0ca862f293b98b8ba3:1
9b812a05907a80b9740d2d92c02f76a7782e00c2dc8c5bfeb2006f569732cf97:1
bf9e4de0fccc260981faec263bf3b4ad321ea519a6ee268d59f3df04c89fdda8:1

Thank you !

**Comment by sputn1ck** [comment]
Hi, 

did you run the `restorechanbackup` command? If you did, this will make lnd reach out to your peers and ask them to force close your channels, so all closures are expected (see: https://docs.lightning.engineering/lightning-network-tools/lnd/disaster-recovery#b-static-channel-backup). As for the `waiting close` status, when did you run the command? A force close can take up to 2 weeks.

**Comment by Nisaba** [comment]
Hi,
I've used the RTL UI to restore backup.
I'm OK with the channel that has been force closed.
For the waiting close channels, I didn't close them myself. I found them in waiting status, I don't know why, probably initiated by the counterparty
As I speak to you, I see that I have collected onchain funds from a channel. There are now 2 left in waiting close status, we can assume that the rest will come...
But I have 2 questions:
- Why have  1 channel moved to force close and 3 other channels moved to closed status ? Who or what is behind these closures ?
- Waiting more than 2 weeks for waiting close channels is it normal ?

Thank you for your precisions !

**Reply by sputn1ck** [reply]
> Why have 1 channel moved to force close and 3 other channels moved to closed status ? Who or what is behind these closures ?

If you recover from the backup you signal you peers that they close the channel. AFAIU they will try to cooperatively close first, but otherwise they will force close.

> Waiting more than 2 weeks for waiting close channels is it normal ?

A force close shouldn't be much longer than 2 weeks, but channels can have different timeout lengths.

**Comment by Nisaba** [comment]
And also, the question about safe moving a LN node from a server to another is still unaswered for me...

**Reply by sputn1ck** [reply]
This is the relevant doc for migrating https://docs.lightning.engineering/lightning-network-tools/lnd/migrating-lnd
